### PR TITLE
Update colour palette and add logo

### LINKED ISSUE #816: Put the logo in and update the palette

### PR DESCRIPTION
**Pull request by darrenvong**
Screenshot 2021-03-24 at 18 41 54

Screenshot 2021-03-24 at 18 42 10

Screenshot 2021-03-24 at 18 42 42

Screenshot 2021-03-24 at 18 42 51

Closes #816 once merged.

### CODE REVIEW

File: app/frontend/src/components/Navigation/Navigation.tsx [hunk 1]
```diff
@@ -56,17 +58,12 @@ const useStyles = makeStyles((theme) => ({
     top: 0,
   },
   flex: {
-    [theme.breakpoints.up("md")]: {
```
Comment (darrenvong) [inline app/frontend/src/components/Navigation/Navigation.tsx:59]: Some redundant styles here now that we are using the `Hidden` helper

File: app/frontend/src/components/Navigation/Navigation.tsx [hunk 2]
```diff
@@ -91,7 +92,13 @@ const useStyles = makeStyles((theme) => ({
     paddingRight: 0,
   },
   nav: {
+    alignItems: "center",
```
Comment (darrenvong) [inline app/frontend/src/components/Navigation/Navigation.tsx:92]: Safari stretches the nav bar height by default without this. Not sure why other browsers don't do this as default value for `align-items` is `stretch` so technically Safari was right... 🤷

File: app/frontend/src/components/Navigation/Navigation.tsx [hunk 3]
```diff
@@ -91,7 +92,13 @@ const useStyles = makeStyles((theme) => ({
...
     display: "flex",
+    flex: 0,
```
Comment (darrenvong) [inline app/frontend/src/components/Navigation/Navigation.tsx:94]: Firefox makes the left nav bar wider than it needs to be without this. Again not sure why other browsers don't do this. Luckily with this combo, it works on all of them now. The fun of dealing with inconsistencies in the frontend...

File: app/frontend/src/resources/logo.svg [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 150 150">
+  <g fill="#E47701">
```
Comment (darrenvong) [inline app/frontend/src/resources/logo.svg]: If I didn't check this on Safari (other browsers happily hide the other redundant bits with `<switch>` for some reason in the original SVG...), didn't even realise how much crap there were in the original logo, including duplicates of it that we don't need!

**Review by darrenvong — COMMENTED** [review]
Lol this issue definitely took longer than expected due to all these weird things I stumbled along the way 🙃

**Review by lucaslcode — DISMISSED** [review]
Should we remove all instances of button `color="secondary"`?

**Comment by aapeliv** [comment]
The favicon is too small :)

Screen Shot 2021-03-24 at 5 19 33 pm

**Comment by darrenvong** [comment]
Updated with all `color="secondary"` I can find + larger favicon

**Review by lucaslcode — APPROVED** [review]

**Pull request merged**